packages/workspace/src/generators/new/new.ts

```diff
@@ -130,7 +130,12 @@ async function initializeGitRepo(
     );
     return;
   }
-  await execute(['init']);
+  if (gte(gitVersion, '2.28.0')) {
+    await execute(['init', '-b', deduceDefaultBase()]);
```

Comment:
This is weird that this does not consider `options.defaultBase`. I know it didn't before but could you add it here? :thinking: